R/Italy.R

```diff
@@ -0,0 +1,72 @@
+#' R6 Class containing country specific attributes and methods
+#'
+#' @description Country specific information for downloading, cleaning
```

Comment:
Add info on where the data comes from back in (I realise its in the code but helpful to have in docs and human text)